pkgs/sketch_pad/lib/editor/editor.dart

```diff
@@ -377,37 +398,37 @@ enum CompletionType {
 }
 
 extension CompletionSuggestionExtension on services.CompletionSuggestion {
-  HintResult toHintResult() {
-    var altDisplay = completion;
-    if (elementKind == 'FUNCTION' ||
-        elementKind == 'METHOD' ||
-        elementKind == 'CONSTRUCTOR') {
-      altDisplay = '$altDisplay()';
-    }
-
-    return HintResult(
-      completion,
-      displayText: displayText ?? altDisplay,
-      className: this.deprecated ? 'deprecated' : null,
-    );
-  }
+  // HintResult toHintResult() {
```

Comment:
This is used when we want to convert an analysis server code completion to a codemirror HintResult.